src/index.html

```diff
@@ -1,13 +1,73 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0
+      ">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <h1>Product cards</h1>
+
+    <div class="card" data-qa="card">
+      <div class="card__photo">
+        <img
+          class="photo"
+          src="./images/imac.jpeg"
+          alt="photo"
+        >
+      </div>
+
+      <h2 class="card__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h2>
+
+      <p class="card__code">
+        Код товара: 195434
+      </p>
+
+      <div class="card__review">
+        <div class="stars stars--4">
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+        </div>
+
+        <p class="card__countReview">
+          Отзывов: 5
+        </p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__priceText">
+          Цена:
+        </p>
+
+        <p class="card__priceN">
```

Comment:
Make class names more readable
```suggestion
        <p class="card__priceNum">
```